cryptoapis/adapter.js

```diff
@@ -38,31 +39,32 @@ const price = (jobRunID, input, callback) => {
     })
 }
 
-const difficultyParams = {
+const latestBlockParams = {
   blockchain: ['blockchain', 'coin'],
+  endpoint: false,
+  q: false,
   network: false,
 }
 
-const difficulty = (jobRunID, input, callback) => {
-  const validator = new Validator(input, difficultyParams)
+const latestBlock = (jobRunID, input, callback) => {
```

Comment:
I like how you renamed this as `latestBlock`, as this actually is the `latestBlock` endpoint from the `cryptoapis` data provider. We also call our input param `endpoint`, but IMHO this is the wrong name for this input param.

What our input param should be called is a `dataPoint`, which we then map to 1 (or more) underlying API endpoints from which we fetch data.

Here the `cryptoapis` adapter supports two data points (we call them `endpoint`) and for both of those (difficulty|height) we fetch data from the same underlying endpoint (latestBlock).